### ISSUE TITLE
Sign in to confirm you’re not a bot error message in Encoder

### ISSUE THREAD

**Issue by AViSource** [WWBN/AVideo#9241]
Since last night and continuing today users including admin are not able to share upload / encode Youtube videos. This error appears:

"Sorry!
ERROR: [youtube] 6R17SNKEbpY: Sign in to confirm you’re not a bot. This helps protect our community. Learn more. "

However, the "Learn more" does not have a hyperlink. And I have not been able to find an answer or solution from my searches online. Has anyone experienced this same error and know what's the cause and solution?

Thanks

**Comment by DanielnetoDotCom** [comment]
Thank you for bringing the issue with uploading and encoding YouTube videos to my attention. 

Since last night, users, including admins, have encountered the following error when attempting to share, upload, or encode YouTube videos:

```
Sorry!
ERROR: [youtube] 6R17SNKEbpY: Sign in to confirm you’re not a bot. This helps protect our community. Learn more.
```

This issue is a global problem related to `yt-dlp`, which is currently being blocked by YouTube. Unfortunately, there is no solution available at the moment.

I am actively following the issue for a solution, and you can track the progress and updates on this new issue at the following link: [GitHub Issue #9242](https://github.com/WWBN/AVideo/issues/9242).

Thank you for your understanding and patience as we work towards resolving this problem.

**Comment by eitanwa** [comment]
So just to be sure about it
Because this is a very important matter
So that we won't be blocked either
There is nothing we can prevent such as any updates
It's just a matter of luck
I understand it right?
Thanks Daniel

**Comment by DanielnetoDotCom** [comment]
We depend on yt-dlp to download from youtube. the solution for it must come from them .

**Comment by eitanwa** [comment]
Thanks Daniel

**Comment by AViSource** [comment]
Actually I didn't know this subject is now determined to be stale. Has a solution been found for the problem:

"Thank you for bringing the issue with uploading and encoding YouTube videos to my attention.

Since last night, users, including admins, have encountered the following error when attempting to share, upload, or encode YouTube videos:

Sorry!
ERROR: [youtube] 6R17SNKEbpY: Sign in to confirm you’re not a bot. This helps protect our community. Learn more.
This issue is a global problem related to yt-dlp, which is currently being blocked by YouTube. Unfortunately, there is no solution available at the moment.

I am actively following the issue for a solution, and you can track the progress and updates on this new issue at the following link: https://github.com/WWBN/AVideo/issues/9242.

Thank you for your understanding and patience as we work towards resolving this problem."
